docs/guidelines/platforms/android/guideline_percievable_android.md

```diff
@@ -63,6 +63,39 @@ class CaptionUtils(context: Context) {
 }
 ```
 
+### Audio Description or Media Alternative (WCAG 1.2.3 - Level A)
+
+This guideline covers point [1.2.3 Audio Description or Media Alternative (Prerecorded) - Level A](https://www.w3.org/WAI/WCAG22/quickref/#audio-description-or-media-alternative-prerecorded) of the WCAG standard.
+
+People with vision impairments may have difficulty understanding key visual details in a video. To make the video accessible to everyone, the guideline requires providing an audio description or an alternative method to convey the visual information.
+
+:white_check_mark: **Success criteria**
+
+To satisfy the guideline, there are two ways to make synchronized media accessible for users with vision impairments:
+* **Provide an audio description**: Add narration to describe key visual elements (e.g., actions, scenery) in the video that someone with vision impairments would otherwise miss.
+* **Provide a text alternative**: If audio description isn't possible, offer a transcript that includes descriptions of both the audio and visual content, allowing users to read what happens in the video.
```

Comment:
There isn’t a specific guideline detailing which videos require alternatives. However, it can be concluded that alternatives are necessary only for important information essential for users to understand the video. This conclusion is supported by the testing procedure, which includes:

1. View the synchronized media presentation while referring to the alternative for time-based media.
2. Check that the dialogue in the alternative for time-based media matches the dialogue in the synchronized media presentation.
3. Check that the alternative for time-based media has descriptions of sounds.
4. Check that the alternative for time-based media has descriptions of setting and setting changes.
5. Check that the alternative for time-based media has descriptions of actions and expressions of any 'actors' (people, animals etc).
6. If the alternate version(s) are on a separate page, check for the availability of link(s) to allow the user to get to the other versions.

For the HomeID highlights, I believe they don’t need alternatives since these videos are purely decorative and designed to catch users' attention visually, lacking any important information.